etl/tests/conftest.py

```diff
@@ -102,41 +103,45 @@ def sample_fixture() -> str:
     return os.path.join(os.path.dirname(__file__), 'sample.csv')
 
 
-@pytest.fixture
-def azure_container_name() -> str:
-    return 'test-container'
+def _get_blob_service(coords: transform.AzureCoordinates)-> blob.BlockBlobService:
+    return blob.BlockBlobService(account_name=coords.account,
+                                 account_key=coords.key,
+                                 custom_domain=coords.domain)
 
 
 @pytest.fixture
-def azure_service(azure_container_name: str,
-                  monkeypatch: _pytest.monkeypatch.MonkeyPatch) -> typing.Iterator[blob.BlockBlobService]:
+def azure_coordinates(monkeypatch: _pytest.monkeypatch.MonkeyPatch) -> transform.AzureCoordinates:
     if not azure_emulator_is_running():
         pytest.skip("Azure emulator is not running.")
 
     account = 'devstoreaccount1'
     key = 'Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw=='
     domain = 'http://127.0.0.1:10000/devstoreaccount1'
+    container_name = 'test-container'
     monkeypatch.setenv('EXTRACT_ENDPOINT_STORAGE_ACCOUNT', account)
     monkeypatch.setenv('EXTRACT_ENDPOINT_STORAGE_KEY', key)
     monkeypatch.setenv('EXTRACT_ENDPOINT_STORAGE_DOMAIN', domain)
-    monkeypatch.setenv('EXTRACT_ENDPOINT_CONTAINER', azure_container_name)
+    monkeypatch.setenv('EXTRACT_ENDPOINT_CONTAINER', container_name)
+    return transform.AzureCoordinates(key=key, account=account, container=container_name, domain=domain)
+
 
-    azure_service = blob.BlockBlobService(account_name=account,
-                                          account_key=key,
-                                          custom_domain=domain)
-    azure_service.create_container(azure_container_name)
-    yield azure_service
-    azure_service.delete_container(azure_container_name)
+@pytest.fixture
+def azure_service(azure_coordinates: transform.AzureCoordinates) -> typing.Iterator[transform.AzureCoordinates]:
```

Comment:
It's a bit confusing that `azure_service` returns `AzureCoordinates` rather than a `BlockBlobService`. Could this be renamed? even something like `azure_emulator` maybe?